impl/src/rendering/templates/remote_crates.bzl.template

```diff
@@ -18,6 +18,11 @@ def {{workspace.gen_workspace_prefix}}_fetch_remote_crates():
         commit = "{{crate.source_details.git_data.commit}}",
         build_file = Label("{{workspace.workspace_path}}/remote:BUILD.{{crate.pkg_name}}-{{crate.pkg_version}}.bazel"),
         init_submodules = True,
```

Comment:
Looks like this line needs to be removed.

Reply:
Fixed.